test/unit/region.spec.js

```diff
@@ -247,6 +246,22 @@ describe('region', function() {
       expect(this.viewOnBeforeShowSpy).to.have.been.calledWith(this.view, this.myRegion, this.showOptions);
     });
 
+    describe('Render and show event ordering', function() {
+      beforeEach(function() {
+        this.viewEvents = _.map(this.triggerSpy.getCalls(), function(call) {
+          return call.args[0];
+        });
+      });
+
+      it('triggers the before render event before the before show event', function() {
+        expect(_.indexOf(this.viewEvents, 'before:render')).to.be.below(_.indexOf(this.viewEvents, 'before:show'));
```

Comment:
I'll give it a shot. I tried that when I was first writing these and was having trouble getting it to work properly.